terraform/modules/check-user-expiry-lambda/lambda/lambda_function.py

```diff
@@ -109,19 +126,25 @@ def extract_email_from_user_attributes(user):
 def send_email(email_from, email_to, email_subject, email_body):
     CHARSET = "UTF-8"
     ses.send_email(
-        Destination={"ToAddresses": [email_to,],},
+        Destination={"ToAddresses": [email_to, ], },
         Message={
-            "Body": {"Html": {"Charset": CHARSET, "Data": email_body,},},
-            "Subject": {"Charset": CHARSET, "Data": email_subject,},
+            "Body": {"Html": {"Charset": CHARSET, "Data": email_body, }, },
+            "Subject": {"Charset": CHARSET, "Data": email_subject, },
         },
         Source=email_from,
     )
 
 
 def lambda_handler(event, context):
-    items = query_dynamodb_users_about_expire()
-    if items:
-        print("Query returned " + str(len(items)) + " item(s)")
-        process_items(items)
+    about_to_expire = query_dynamodb_users_about_expire()
+    if about_to_expire:
+        print("Query returned " + str(len(about_to_expire)) + " item(s)")
+        process_items(about_to_expire, reminder_subject_line, "default_email_template_analytical.html")
+    else:
+        print("Query did not return any due to expire items.")
+    expired = query_dynamodb_users_about_expire()
```

Comment:
Arent you calling the wrong function here, shouldnt it be query_dynamodb_users_expired_today?